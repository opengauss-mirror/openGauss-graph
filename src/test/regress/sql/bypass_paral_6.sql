--
-- bypass parallel test part6
--

start transaction;
select pg_sleep(1);
select * from bypass_paral where col1=1 and col2 is null;
update bypass_paral set col3='pp4' where col2 is null and col1=1;
select * from bypass_paral where col1=1 and col2 is null;
commit;
