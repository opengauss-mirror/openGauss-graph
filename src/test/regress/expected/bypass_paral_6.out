--
-- bypass parallel test part6
--
start transaction;
select pg_sleep(1);
 pg_sleep 
----------
 
(1 row)

select * from bypass_paral where col1=1 and col2 is null;
 col1 | col2 |    col3     
------+------+-------------
    1 |      | test_insert
(1 row)

update bypass_paral set col3='pp4' where col2 is null and col1=1;
select * from bypass_paral where col1=1 and col2 is null;
 col1 | col2 | col3 
------+------+------
(0 rows)

commit;
